chia/wallet/notification_store.py

```diff
@@ -44,6 +46,11 @@ async def create(
                 "CREATE TABLE IF NOT EXISTS notifications(" "coin_id blob PRIMARY KEY," "msg blob," "amount blob" ")"
             )
 
+            try:
+                await conn.execute("ALTER TABLE notifications ADD COLUMN height bigint DEFAULT 0")
+            except sqlite3.OperationalError:
+                pass  # ignore what is likely Duplicate column error
```

Comment:
I would feel better if this was checked specifically for the error we want to ignore, and any other errors were re-thrown.

I suspect we also don't have any automates tests exercising this case, which makes it extra risky to ignore errors

Reply:
I checked for a string in the error message does that work?